batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -0,0 +1,55 @@
+package com.lftechnology.batch7crud.controller;
+
+import static com.lftechnology.batch7crud.constant.MessageConstant.*;
+import static com.lftechnology.batch7crud.constant.AttribConstants.*;
+import static com.lftechnology.batch7crud.constant.UrlConstants.*;
+import static com.lftechnology.batch7crud.constant.ParamConstants.*;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.io.File'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)